machine-learning/app/models.py

```diff
@@ -1,117 +0,0 @@
-import torch
-from insightface.app import FaceAnalysis
-from pathlib import Path
-import os
-
-from transformers import pipeline, Pipeline
-from sentence_transformers import SentenceTransformer
-from typing import Any
-import cv2 as cv
-
-cache_folder = os.getenv("MACHINE_LEARNING_CACHE_FOLDER", "/cache")
-device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
Did you remove cuda support ?

Reply:
This PR doesn't have CUDA support. The intent is to add CUDA through ONNX Runtime instead of PyTorch, so 'device' isn't necessary. PyTorch would be CPU-only and used to export the model to ONNX.

The cache directory having a device folder was to avoid scenarios where an ONNX model that's optimized for CUDA is loaded without CUDA. The current iteration optimizes at runtime so this isn't necessary anymore.

Reply:
But thinking over it, I can reconsider that approach depending on how the rebase on this PR goes. I want to keep the dependencies simple and Poetry struggles with PyTorch. At the same time, some dependencies aren't available through Conda/Mamba.